test/json/test-core.json.d.ts

```diff
@@ -0,0 +1,21 @@
+export type TestMessage = {
+  /** The MF2 syntax source. */
+  src: string;
+  /** The locale to use for formatting. Defaults to 'en-US'. */
```

Comment:
Did you consider defaulting to `und`? The root locale is US English-like but also somewhat invariant.

Reply:
Alas, JS doesn't support an `und` locale.

Reply:
It works in Chrome. It also works in FF, except that the locale's name data isn't present.

Reply:
It only looks like it works, though. If you try this, for instance, you'll get your default locale (probably `en-US`) rather than `und`:
```js
new Intl.NumberFormat('und').resolvedOptions().locale
```